spec/unit/knife/bootstrap_spec.rb

```diff
@@ -1072,7 +1076,7 @@ def certificates
                 keys_only: false, # implied false from config password present
                 key_files: ["/identity.pem", "/gateway.pem"], # Config
                 sudo: true, # ccli
-                verify_host_key: nil, # Config
+                verify_host_key: false, # Config
```

Comment:
This is where the API has been fixed to be correct, the spec here was testing buggy behavior